app/routes/_extras.resources._index/data.server.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  getAllResources,
+  type Category,
+  type Resource,
+} from "~/lib/resources.server";
+import { categories } from "./ui";
+import { octokit } from "~/lib/github.server";
+
+export async function getResourcesForRequest(request: Request) {
+  let resources = await getAllResources({ octokit });
+  return {
+    resources,
+    featuredResource: null,
+  };
```

Comment:
This used to have all the `selectedCategory`/`selectedTags` stuff - moved to a `clientLoader` because SSG won't have query params

Reply:
Haha yeah, going back to the server for this doesn't even super make sense, since we have to send down all the data anyway and it's not changing in between deploys

Cool that `await serverLoader();` just grabs whatever the prerender phase created. Never really thought about it but makes total sense